styles/style.css

```
@@ -0,0 +1,26 @@
+html {
+    font-size: 20px;
+    font-family: 'Cormorant', serif;
+  }
+
+  h1 {
+    font-size: 60px;
+    text-align: center;
+    font-family: 'Monsieur La Doulaise', cursive;
+    margin: 0;
+    padding: 20px 0;
+    color: #191310;
+  }
+
+  body {
+    width: 600px;
+    margin: 0 auto;
+    background-color: #F4EAE1;
+    padding: 0 20px 20px 20px;
```

nit: I think if you do 3 values, the 20px will middle value applies to both left and right

```suggestion
    padding: 0 20px 20px;
```